test/smoke/quorum.js

```diff
@@ -194,7 +194,7 @@ describe('Quorums', () => {
       // Wait two seconds here before checking for IS locks
       // TODO: implement this.slow() and await IS ZMQ message to mark test response speed yellow/red
       await new Promise((resolve) => {
-        setTimeout(resolve, 2000);
+        setTimeout(resolve, 4000);
```

Comment:
4 seconds may not be enough. ISLocks can sometimes take longer. Probably up to 10 seconds is reasonable. Additionally your comment is now incorrect

Reply:
Good point. It's a trade-off with waiting because if a block is mined in those 10s, then the test will fail as well. I think implementing ZMQ here is probably a better way to design the test.